lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -51,7 +52,8 @@ abstract class _$ConnectPropsMixin implements UiProps {
   @override
   Map get props;
 
-  dynamic Function(dynamic action)? dispatch;
+  @disableRequiredPropValidation
```

Comment:
I think a comment here would be helpful
```suggestion
  // Disable validation since this prop is set by the `connect` HOC, and does not need to be set by consumers.
  @disableRequiredPropValidation
```